### PR TITLE
fix: validate response

### PR DESCRIPTION
**Pull request by toteto**

### CODE REVIEW

File: libs/ts-rest/core/src/lib/client.ts [hunk 1]
```diff
@@ -215,6 +215,7 @@ export const fetchApi = ({
     rawBody: body,
     rawQuery: query,
     contentType: undefined,
+    validateResponse: clientArgs.validateResponse,
```
Comment (toteto) [inline libs/ts-rest/core/src/lib/client.ts:218]: @Gabrola maybe check if there are other parameters that are forgotten?

File: apps/docs/docs/core/core.md [hunk 2]
```diff
@@ -346,3 +346,32 @@ const parentContract = c.router(
   }
 );
 ```
+
+### Validate Response
```
Comment (Gabrola) [inline apps/docs/docs/core/core.md:350]: This should be part of the `core/fetch` page instead

File: .changeset/beige-forks-sit.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@ts-rest/core': minor
```
Comment (Gabrola) [inline .changeset/beige-forks-sit.md:2]: Patch release

File: .changeset/beige-forks-sit.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+fix client not validating response
```
Comment (Gabrola) [inline .changeset/beige-forks-sit.md:5]: We capitalize these because they are part of the changelog

**Review by Gabrola — CHANGES_REQUESTED** [review]
Looks good, just some minor comments

File: apps/docs/docs/core/core.md (hunk 2 above)
Comment (toteto) [reply apps/docs/docs/core/core.md]: Done b8fa3e8

File: .changeset/beige-forks-sit.md (hunk 4 above)
Comment (toteto) [reply .changeset/beige-forks-sit.md]: Done 0d0af4f

File: .changeset/beige-forks-sit.md (hunk 3 above)
Comment (toteto) [reply .changeset/beige-forks-sit.md]: Done 0d0af4f

**Review by Gabrola — APPROVED** [review]
LGTM! Just made some small changes to the docs

**Pull request merged**